package.json

```diff
@@ -4,7 +4,8 @@
     "description": "A system that allows efficient, cost-effective management of our library",
     "main": "index.js",
     "scripts": {
-        "start": "node server/index.js",
+        "build": "babel ./server -d build",
```

Comment:
build into a folder called `dist`.

Reply:
I have corrected that, please review this again.